packages/contract-tests/test/lib/utils/ens.js

```diff
@@ -25,14 +25,26 @@ const methodABI = {
   stateMutability: "pure",
   type: "function"
 };
-const args = ["my.ens.name", 555, { from: "the.other.name" }];
+const txParams = {
+  from: "the.other.name",
+  accessList: [{
```

Comment:
Is this how you specify this? Do you have links to any docs? I looked at the web3 docs and they kind of ambiguously say to spcify "a list of addresses and storage keys that the transaction plans to access".

Reply:
Yeah, unfortunately, it's not really documented anywhere; I basically just had to consult the code of `@ethereumjs/tx` to see how it's supposed to work.  There are two allowed forms:

1. An array of `{ address, storageKeys }` objects.  In this case, `address` is a string (being, y'know, an address)` and `storageKeys` is an array of strings (each being a hex string describing 32 bytes).
2. An array of `[address, storageKeys]` tuples.  In this case, `address` is a Uint8Array, and `storageKeys` is an array of Uint8Arrays.

As the comment I wrote mentions, I decided that there's not much reason to handle the latter. :P

Reply:
Here's where the type is actually defined: https://github.com/ethereumjs/ethereumjs-monorepo/blob/c74035e12765ab4bce8cb2bead634e2887af0d7c/packages/tx/src/types.ts#L80

Reply:
OK, it turns out I was wrong about the second form being allowed.  I'll go delete that comment, I guess!

Reply:
Thanks @haltman-at!